.docker/nginx.conf

```diff
@@ -0,0 +1,21 @@
+server {
+    listen 8080;
+    server_name localhost;
+    root /usr/share/nginx/html;
+
+    gzip on;
+    gzip_comp_level 5;
+    gzip_min_length 256;
+    gzip_proxied any;
+    gzip_vary on;
+    gzip_types
+      application/javascript
+      image/svg+xml
+      text/css
+      text/json
+    ;
+
+    location ~* ^(/.+\.bundle\..+)|(/static/media/.+)$ {
```

Comment:
Would be useful to know if there's a way to isolate Webpack-created files (ie with a version in the name); looks like Storybook doesn't let you touch this part of the Webpack config.

Reply:
There is a manifest plugin that will report out the assets created by Webpack, that could be used?

Reply:
Sounds tricky to make use of. Ideally would just move them all to a specific subdirectory...